java/README.md

```diff
@@ -245,6 +261,21 @@ public class Main {
 }
 ```
 
+### Scala and Kotlin Examples
+See [our Scala and Kotlin examples](../examples/) to learn how to use Valkey GLIDE in Scala and Kotlin projects.
+
+#### Scala
+To run the Scala example, from the `examples/scala/example` directory, execute:
+```shell
+sbt run
+```
+
+#### Kotlin
+To run the Kotlin example, from the `examples/kotlin/example` directory, execute:
+```shell
+./gradlew run
+```
```

Comment:
Please remove it from here and add README file to each example folder that explains how to run the example